app/services/dunning_campaigns/update_service.rb

```diff
@@ -45,21 +32,84 @@ def call
 
     attr_reader :dunning_campaign, :organization, :params
 
+    def update_dunning_campaign_attributes
```

Comment:
do we really need this method?

i mean, we can directly call `assign_attributes`, no? :)

Reply:
true, it does not add much value. I'll change it.